README.md

```diff
@@ -107,7 +107,27 @@ If at some point you see something that could be better, then please open a PR.
 
 `pacman -S --needed base-devel git arch-install-scripts repoctl fuse-overlayfs rsync python-telegram-send`
 
-One must have an active mirror of chaotic-aur running locally and some signing key. Configure them in `/etc/chaotic.conf`, like this:
+One must have an active mirror of chaotic-aur running locally and some signing key. 
+To create a gpg key & ssh keypair for root user do the following (chowning tty is needed as the process fails otherwise):
+
+```sh
+sudo su -l root
+chown root $(tty)
+gpg --full-generate-key
+ssh-keygen
+cat /root/.ssh/id_rsa.pub
+```
+
+The ssh public key (output of cat /root/.ssh/id_rsa.pub) then needs to be added to the primary servers root authorized keys (/root/.ssh/authorized_keys)
+Get the gpg key signature & armored public key by running:
+
+```sh
```

Comment:
Replace this snippet with a link to https://wiki.archlinux.org/index.php/GnuPG#Export_your_public_key